cfme/fixtures/tag.py

```diff
@@ -119,6 +120,10 @@ def _check_item_visibility(vis_object, vis_expect):
                 vis_object.remove_tag(tag=tag)
         with user_restricted:
             try:
+                if isinstance(vis_object, Host):
+                    # need to remove the link to the provider from the host,
+                    # so the navigation goes Compute -> Infrastructure -> Hosts, not Providers
+                    vis_object.parent.filters.update({'provider': None})
```

Comment:
:+1: 

@mshriver we are getting this diffraction due to use of `collection.filter` (provider key) object in navigation. I think we have to take look at it.